src/mngt/ocf_mngt_cache.c

```diff
@@ -462,10 +462,13 @@ static void _ocf_mngt_load_add_cores(ocf_pipeline_t pipeline,
 
 		length = ocf_volume_get_length(&core->volume);
 		if (length != core->conf_meta->length) {
-			ocf_cache_log(cache, log_err,
+			ocf_core_log(core, log_err,
 					"Size of core volume doesn't match with"
 					" the size stored in cache metadata!");
-			goto err;
+		
+			_ocf_mngt_close_all_uninitialized_cores(cache);
```

Comment:
this function has been very recently changed to accept "context" parameter rather than cache.